portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -423,3 +425,57 @@ function drawInteractiveChart() {
         interactiveChart.draw(emotionData, options);
       });
 }
+
+const typedText = document.querySelector('.typed-text');
+/**
+ * TODO Stretch Feature(briafassler): read names from a database instead of an
+ * array
+ */
+const martyrNames = [
+  'George Floyd',          'Breonna Taylor',  'Ahmaud Arbery',
+  'Rayshard Brooks',       'Robert Fuller',   'James Scurlock',
+  'Elijah McClain',        'Justin Howell',   'Jamel Floyd',
+  'Eric Garner',           'Michael Brown',   'Eric Reason',
+  'Sandra Bland',          'Jamar Clark',     'Tamir Rice',
+  'John Crawford III',     'Phillip White',   'Ezell Ford',
+  'Dante Parker',          'Laquan McDonald', 'Dominique Clayton',
+  'Michelle Cusseaux',     'George Mann',     'Tanisha Anderson',
+  'Akai Gurley',           'Alonzo Smith',    'Stephon Clark',
+  'Philando Castile',      'Janet Wilson',    'Darrius Stewart',
+  'Botham Jean',           'Samuel Dubose',   'Willie Tillman',
+  'Quintonio Legrier',     'Brian Keith Day', 'Christian Taylor',
+  'Junior Prosper',        'Freddie Gray',    'Christopher McCorvey',
+  'Peter Gaines',          'Mya Hall',        'Felix Kumi',
+  'Keith Harrison McLeod', 'Anthony Ashford', 'Salvado Ellswood',
+];
+let marytrNamesIndex = 0;
+let charIndex = 0;
```

Comment:
is there a reason these variables have a global scope? it seems unneccesary

Reply:
they've now been wrapped in a function